.prettierrc

```diff
@@ -11,10 +11,11 @@
         "add-field-many.graphql",
         "inline-many.graphql",
         "inline-many-list.graphql",
-        "test-grpc.graphql"
+        "test-grpc.graphql",
+        "test-expr-success.graphql"
       ],
       "options": {
-        "printWidth": 200
+        "printWidth": 216
```

Comment:
One must find creative ways to avoid doing this.

Reply:
imo the test assertions should ignore whitespace instead of doing a straight string comparison. Maybe first minify then compare.